test/suite/test_sweep04.py

```diff
@@ -0,0 +1,225 @@
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# test_sweep04.py
+# Test lots of tables with more steadily created and dropped.
+# A core group of tables is used most often and hangs around.
+#
+
+import time
+from suite_random import suite_random
+from wiredtiger import stat
+import wttest
+
+# Given a set of values corresponding to successive times,
+# we have an implied set of points in two dimensions.
+# Compute the average for the values and the slope for the
+# least squares regression of the line.
+#
+# We'd use numpy, but we aren't assured that it is always installed
+# for our python.
+def average_slope(y):
+    n = len(y)
+    if n == 0:
+        return [0, 0]  # there's no average or slope
+    elif n == 1:
+        return [y[0], 0]  # there's no slope
+
+    average = sum(y) / n
+    x = range(1, n + 1)  # the implied x axis, steadily increasing integers.
+
+    # Here's a formula for least squares slope:
+    #   https://www.mathsisfun.com/data/least-squares-regression.html
+    top = n * sum(x[i] * y[i] for i in range(n)) - sum(x) * sum(y)
+    bottom = n * sum(x[i]**2 for i in range(n)) - sum(x)**2
+    slope = top / bottom
+    return [average, slope]
+
+@wttest.longtest("lots of files")
+class test_sweep04(wttest.WiredTigerTestCase):
+    tablebase = 'test_sweep04'
+    uri = 'table:' + tablebase
+
+    # Configuration values for the run. If any of these values are changed,
+    # the simulation will run, but the acceptance criteria may fail at the end.
+    core_tables = 10               # Number of core tables that always exist
+    transient_tables = 100         # Number of transient tables at any time
+    transient_examined = 10        # Number of transient tables opened at a time
+    ratio_examined = 0.01          # The chance that transient tables are examined
+    transient_table_max = 10000    # defines the length of the run
+    numkv = 1                      # Number of k/v pairs. Shouldn't matter for this test.
+    nsessions = 100                # Number of sessions in our pool.
+
+    conn_config = 'file_manager=(close_handle_minimum=0,' + \
```

Comment:
Since this is a sweep test, the values in here probably matter. A comment describing them and their impact would be helpful.